project/Versions.scala

```diff
@@ -21,7 +21,7 @@ object Versions {
   // and install in a local Maven repository. This is all done automatically, however it will work
   // only on Unix/OSX operating system. Windows users have to build and install Spark manually if the
   // desired version is not yet published into a public Maven repository.
-  val ApacheSpark     = "3.2.1"
+  val ApacheSpark     = "3.3.1"
   val SparkJetty      = "9.3.27.v20190418"
```

Comment:
I think we can uplift the jetty version to align with the [3.3.1 tagged source](https://github.com/apache/spark/releases/tag/v3.3.1)
```suggestion
  val SparkJetty      = "9.4.48.v20220622"
```